mangaki/mangaki/views.py

```diff
@@ -900,6 +944,7 @@ def update_evidence(request):
     needs_help = request.POST.get('needs_help') == 'True'
     delete = request.POST.get('delete')
     suggestion_id = request.POST.get('suggestion')
+    redirect_link = request.POST.get('redirect')
```

Comment:
Usually, we use `next` for links where we will be redirected.
(`/login?next=/animes` for example)